content/docs/command-reference/config.md

```diff
@@ -27,15 +27,20 @@ This command reads and updates the DVC configuration files. By default, the
 regular project's config file in `.dvc/config` is read or modified. This file is
 meant to be tracked by Git and should not contain sensitive and/or user-specific
 information (passwords, SSH keys, etc). Use the `--local` command option (flag)
-instead, to set (or override) secrets. `--global` and `--system` are also
-available to set config options for multiple projects and users, respectively.
-
-| Command option | Priority | Linux location         | Windows location                                          |
-| -------------- | -------- | ---------------------- | --------------------------------------------------------- |
-| `--local`      | 1        | `.dvc/config.local`    | `.dvc/config.local`                                       |
-| None (default) | 2        | `.dvc/config`          | `.dvc/config`                                             |
-| `--global`     | 3        | `~/.config/dvc/config` | `%LocalAppData%/iterative/dvc/config`                     |
-| `--system`     | 4        | `/etc/dvc.config`      | `%AllUsersProfile%/Application Data/iterative/dvc/config` |
+instead, to set (or override) secrets:
+
+| Flag           | Priority | Config file location |
+| -------------- | -------- | -------------------- |
+| `--local`      | 1        | `.dvc/config.local`  |
+| None (default) | 2        | `.dvc/config`        |
+
+The `--global` and `--system` flags are also available to set config options for
+multiple projects and users, respectively:
+
+| Flag                                                | Priority | Mac location                            | Linux location             | Windows location                                          |
+| --------------------------------------------------- | -------- | --------------------------------------- | -------------------------- | --------------------------------------------------------- |
+| <span style="white-space: nowrap">`--global`</span> | 3        | `$HOME/Library/Application Support/dvc` | `$HOME/.config/dvc/config` | `%LocalAppData%/iterative/dvc/config`                     |
+| <span style="white-space: nowrap">`--system`</span> | 4        | `/Library/Application Support/dvc`      | `/etc/dvc.config`          | `%AllUsersProfile%/Application Data/iterative/dvc/config` |
```

Comment:
Notice the ugly HTMl needed so that `--` isn't separated from `global/system`. I tried a bunch of MD tricks and couldn't fix this otherwise... Any ideas anyone? Cc @rogermparent

Reply:
p.s. this is how it looks right now:

![image](https://user-images.githubusercontent.com/1477535/93561889-97544580-f94a-11ea-8697-538f1f55170e.png)
> From https://dvc.org/doc/command-reference/config

Reply:
You can use this snippet in the Markdown to apply this style page-wide without the ugly in-table repeats.

```
<style>
  #markdown-root td:first-child code {
    white-space: nowrap;
  }
</style>
```

This also removes the risk of collision if any other table depends on having a code block in the first column wrap.

Reply:
OK thanks Roger, I'll use that. It's also not ideal and needs an HTML comment to explain what's happening but it's better.